runtime/altair/src/xcm.rs

```diff
@@ -69,24 +69,17 @@ impl frame_support::traits::Contains<RuntimeCall> for SafeCallFilter {
 	fn contains(call: &RuntimeCall) -> bool {
 		matches!(
 			call,
-			RuntimeCall::System(
-				frame_system::Call::kill_prefix { .. } | frame_system::Call::set_heap_pages { .. },
-			) | RuntimeCall::Timestamp(..)
+			RuntimeCall::Timestamp(..)
 				| RuntimeCall::Balances(..)
-				| RuntimeCall::Session(pallet_session::Call::purge_keys { .. })
-				| RuntimeCall::Treasury(..)
 				| RuntimeCall::Utility(pallet_utility::Call::as_derivative { .. })
-				| RuntimeCall::Vesting(..)
 				| RuntimeCall::PolkadotXcm(
 					pallet_xcm::Call::limited_reserve_transfer_assets { .. }
-				) | RuntimeCall::CollatorSelection(
-				pallet_collator_selection::Call::set_desired_candidates { .. }
-					| pallet_collator_selection::Call::set_candidacy_bond { .. }
-					| pallet_collator_selection::Call::register_as_candidate { .. }
-					| pallet_collator_selection::Call::leave_intent { .. },
-			) | RuntimeCall::XcmpQueue(..)
+				) | RuntimeCall::XcmpQueue(..)
 				| RuntimeCall::DmpQueue(..)
 				| RuntimeCall::Proxy(..)
+				| RuntimeCall::LiquidityPoolsGateway(
```

Comment:
These changes to the other calls are unrelated to this PR and could introduce undesired behaviour given that we don't have tests in place to properly assess this. Not a blocker tho

Reply:
I honestly don't know why this diff still exists. This change was merged yesterday already with #1526. I previously had this PR based against the #1526 branch.